test/test-generate-kmp.ts

```diff
@@ -0,0 +1,29 @@
+import { WordListFromArray } from "@predictive-text-studio/lexical-model-compiler";
+import { generateKmp } from "@worker/generate-kmp";
+
+import test from "ava";
+import JSZip = require("jszip");
+
+const langName = "English";
+const bcp47Tag = "en";
+const sources = [
+  new WordListFromArray("wordlist.xlsx", [
+    ["TŦE", 13644],
+    ["E", 9134],
+    ["SEN", 4816],
+  ]),
+];
+const modelID = "nrc.en.mtnt";
```

Comment:
ID or Id

Reply:
I will use Id to make the variable name consistent.